students/kpersic/module_2/09_company-website/about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="What we do and why we do it">
+    <title>About</title>
+    <style>
+        .header {
+            background-color: rgb(218, 216, 216);
+            padding: 10px;
+            border-style: solid;
+            border-width: 2px;
+        }
+
+        footer {
+            background-color: rgb(218, 216, 216);
+            padding: 10px;
+            border-style: solid;
+            border-width: 2px;
+        }
+
+        main {
+            background-color: rgb(190, 218, 218);
+            padding: 10px;
+        }
+    </style>
+</head>
+
+<body>
+    <!--header-->
+    <header class="header">
+        <div>
+            <img src="logo.svg" alt="Company logo" height="50">
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="products.html">Products</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <!--Main-->
+    <main>
+        <header>
+            <h1> What we do and why we do it</h1>
+        </header>
+
+        <section>
+            <figure>
+                <img src="billy-corgan.jpg" alt="Billy Corgan">
+                <figcaption>Billy Corgan, CEO</figcaption>
+            </figure>
+
+            <p>"Ipsa netus consequatur blanditiis nostrum pretium phasellus sollicitudin, quae ad aut taciti,
```

Comment:
Pogledaj primjer kako rukovati citatima u HTML-u, za to postoji poseban element koji se zove `<blockquote>`.

https://github.com/dstrekelj/algebra-front-end-developer-2019/blob/master/module-2-html/examples/17-structure-semantic/index.html